Player/Player.cs

```diff
@@ -223,9 +223,15 @@ public void Respawn()
 
 
 	[Remote]
-	public void ApplyDamage(float Damage)
+	public void ApplyDamage(float Damage, int AttackerId)
```

Comment:
Rather than grabbing the position from the local Player instance it should instead pass in the position that the bullet came from. This is both more accurate (right now it uses the center of the player) as well as more up to date (the player might have moved between we last got their position and them shooting)